### ISSUE TITLE
error in code

### ISSUE THREAD

**Issue by Jannik44** [fboulnois/stable-diffusion-docker#72]
my command is the following and it throws an error

sudo bash ./build.sh run --token hf_sagfdcnhfxvbgc --device cpu --onnx 'Andromeda galaxy in a bottle'

![IMG_20230224_132253](https://user-images.githubusercontent.com/76906033/221178427-7c7193b4-5239-457d-9e07-1deb32489cb2.jpg)

**Comment by fboulnois** [comment]
Use `--prompt`:

```sh
sudo bash ./build.sh run --token hf_... --device cpu --onnx --prompt 'Andromeda galaxy in a bottle'
```

**Comment by fboulnois** [comment]
Resolved by #74
